benchmarks/compare.py

```diff
@@ -24,14 +24,8 @@
 from typing import Dict, List, Any
 from pathlib import Path
 from argparse import ArgumentParser
-
-try:
```

Comment:
I think this is still a helpful error message

Reply:
Thats a good point and I can roll this back. Since rich isn't STL its probably a good idea to keep this.